app/views/subscription_mailer/send_digest.html.erb

```diff
@@ -1,46 +1,38 @@
 <header style="text-align: center;color:black">
-	<img src="https://avatars2.githubusercontent.com/u/4621650?s=200&v=4" style="width: 100px;height: 100px">
-	<p style="font-size: 20px">[Public Lab] Your Weekly Digest</p>
+  <img src="https://avatars2.githubusercontent.com/u/4621650?s=200&v=4" style="width: 100px;height: 100px">
```

Comment:
This could use code like this: `<% if n.author.photo %><%= n.author.photo_path(:thumb) %><% end %>` but also could be done in a follow-up PR if you like?

Reply:
I think you mean to change in line 16, ie Author's profile image right? This one is for the PL logo, if I am not wrong.

Reply:
Okay, so I have done this, and it works!:

![image](https://user-images.githubusercontent.com/40794215/52517614-ae598e80-2c63-11e9-804c-08baa62527d5.png)